compiler/rustc_mir/src/util/patch.rs

```diff
@@ -117,10 +117,6 @@ impl<'tcx> MirPatch<'tcx> {
         self.add_statement(loc, StatementKind::Assign(box (place, rv)));
     }
 
-    pub fn make_nop(&mut self, loc: Location) {
-        self.make_nop.push(loc);
-    }
```

Comment:
Can you remove the `make_nop` field too?

Reply:
Done.

Reply:
Please leave this in. The mir-patch helper may not be used in its entirety, but it was designed for more general use cases than its initial uses.

Reply:
This was not resolved